frontend/src/containers/03_products/Products.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import axios from "axios";
+
+import "./products.css";
+
+class Products extends React.Component {
+  state = {
+    customerId: 1,
+
+    modal: {
+      display: "none",
+      name: "",
+      disabled: true
+    }
+  };
+
+  changeValue = event => {
+    const { target } = event;
+    const { modal } = this.state;
+
+    let newValue = modal.name === "" ? true : false;
+
+    this.setState({
+      modal: {
+        ...modal,
+        [target.name]: target.value,
+        disabled: newValue
+      }
+    });
+  };
+
+  showPopUp = () => {
+    const { modal } = this.state;
+
+    let newDisplay = "";
+
+    if (modal.display === "none") {
+      newDisplay = "flex";
+    } else {
+      newDisplay = "none";
+    }
+
+    this.setState({
+      modal: {
+        ...modal,
+        display: newDisplay
+      }
+    });
+  };
+
+  removePopUp = () => {
+    const { modal } = this.state;
+
+    this.setState({
+      modal: {
+        ...modal,
+        display: "none"
+      }
+    });
+  };
+
+  addProduct = event => {
+    const { modal, customerId } = this.state;
+
+    event.preventDefault();
+
+    let data = {
+      customerId: customerId,
+      name: modal.name
+    };
+
+    let json = {
+      action: "insertProduct",
+      data: data
+    };
+
+    let config = {
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json, application/x-www-form-urlencoded"
+      }
+    };
+
+    axios
```

Comment:
Moet in een andere bestand (store -> actions) zelfde geld voor de test data enzo

Reply:
Marco zegt dat het niet hoeft voor deze sprint